{{cookiecutter.project_name}}/examples/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.10.2)
+project({{ cookiecutter.module_name }}Examples)
+
+find_package(ITK REQUIRED
+  COMPONENTS
+    ITKImageIO
```

Comment:
This is problematic if the example is built as part of ITK. See [this](https://github.com/InsightSoftwareConsortium/ITKMontage/commit/3cca40a598e359cd28bc15ed5903db9a0100886b).

Reply:
@fbudin69500 do you think we should add this to the module template?

Reply:
This seems difficult to address cleanly for now as the template would have to explicitly instantiate certain IOs and it is not possible to know ahead of time which IOs should be instantiated. There could be hints in the CMakeLists.txt with commented out lines of code with the IO names that the developer could either uncomment or remove, but that doesn't seem ideal. @dzenanz : Was an issue created in the issue tracker for that Meta module IO issue?

Reply:
I just created an issue: https://github.com/InsightSoftwareConsortium/ITK/issues/359

Reply:
We could apply solution from `ITKMontage`:
https://github.com/InsightSoftwareConsortium/ITKMontage/blob/v0.8.2/examples/CMakeLists.txt#L4-L22

Reply:
PR #163.